### ISSUE TITLE
Error with gin path

### ISSUE THREAD

**Issue by xxrraa** [acids-ircam/RAVE#303]
Hi I'm having an error that seems to be related to .gin paths, I'm using ubuntu 20.04.6 LTS.  Tried 6 separate conda envs python 3.9, 3.10, 3.11 x acids-rave 2.3 and 2.3.1, installing torch as instructed on their started locally page, then pip installing acids-rave and conda installing ffmpeg.  I get the following error across all 6 envs.  Previous to the 6 envs I downloaded the latest code from github and ran it that way and got the same error which is why I switched to pip.  Training begins but the error makes me worry that it might not be doing its best/not working at 100% efficiency/accuracy/capacity.  I am about to try the same 6 combinations on windows, but wondering why its not working on Ubuntu, it seems others have had similar issues.  Thanks kindly in advance for any help and for this great release!  

- edit: also tried v1 config flag when training. also tried modifying the train script and explicitly entering the path for the gin file for the version ie v1, v3 and then i didnt get "Path not found" for that gin file but the others still remained

```
I0318 18:11:02.146387 140345576497536 resource_reader.py:50] system_path_file_exists:v3.gin
E0318 18:11:02.146519 140345576497536 resource_reader.py:55] Path not found: v3.gin
I0318 18:11:02.146563 140345576497536 resource_reader.py:50] system_path_file_exists:/home/andrew/anaconda3/envs/r311_2/lib/python3.11/site-packages/rave/v3.gin
E0318 18:11:02.146600 140345576497536 resource_reader.py:55] Path not found: /home/andrew/anaconda3/envs/r311_2/lib/python3.11/site-packages/rave/v3.gin
I0318 18:11:02.146746 140345576497536 resource_reader.py:50] system_path_file_exists:configs/v2.gin
E0318 18:11:02.146812 140345576497536 resource_reader.py:55] Path not found: configs/v2.gin
I0318 18:11:02.147111 140345576497536 resource_reader.py:50] system_path_file_exists:configs/v1.gin
E0318 18:11:02.147179 140345576497536 resource_reader.py:55] Path not found: configs/v1.gin
I0318 18:11:02.155839 140345576497536 resource_reader.py:50] system_path_file_exists:configs/adain.gin
E0318 18:11:02.155908 140345576497536 resource_reader.py:55] Path not found: configs/adain.gin
I0318 18:11:02.156362 140345576497536 resource_reader.py:50] system_path_file_exists:configs/snake.gin
E0318 18:11:02.156426 140345576497536 resource_reader.py:55] Path not found: configs/snake.gin
I0318 18:11:02.157845 140345576497536 resource_reader.py:50] system_path_file_exists:configs/descript_discriminator.gin
E0318 18:11:02.157911 140345576497536 resource_reader.py:55] Path not found: configs/descript_discriminator.gin
/home/andrew/anaconda3/envs/r311_2/lib/python3.11/site-packages/torch/nn/utils/weight_norm.py:28: UserWarning: torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.
  warnings.warn("torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.")
/home/andrew/anaconda3/envs/r311_2/lib/python3.11/site-packages/torchaudio/transforms/_transforms.py:94: UserWarning: `return_complex` argument is now deprecated and is not effective.`torchaudio.transforms.Spectrogram(power=None)` always returns a tensor with complex dtype. Please remove the argument in the function call.
  warnings.warn(
train set: 529 examples
val set: 11 examples
selected gpu: [0]
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name                     | Type                  | Params
-------------------------------------------------------------------
0 | pqmf                     | CachedPQMF            | 16.7 K
1 | encoder                  | VariationalEncoder    | 16.1 M
2 | decoder                  | GeneratorV2           | 15.5 M
3 | discriminator            | DescriptDiscriminator | 42.5 M
4 | audio_distance           | AudioDistanceV1       | 0     
5 | multiband_audio_distance | AudioDistanceV1       | 0     
-------------------------------------------------------------------
74.1 M    Trainable params
0         Non-trainable params
74.1 M    Total params
296.313   Total estimated model params size (MB)
Sanity Checking DataLoader 0: 100%|███████████████| 2/2 [00:00<00:00,  3.21it/s]Computing receptive field for this configuration...
^Z
[1]+  Stopped                 
```

**Comment by xxrraa** [comment]
solved by adding the full path to gin file of the chosen config in train.py under "#parse configuration", and then modifying the chosen config's gin file in "rave/configs" and adding full paths to the gin files included in that gin file, and any gin files included in those gin files
